www/network.js

```diff
@@ -77,15 +63,29 @@ channel.onCordovaReady.subscribe(function() {
         if (channel.onCordovaConnectionReady.state !== 2) {
             channel.onCordovaConnectionReady.fire();
         }
-    },
-    function (e) {
+    }
+
+    function errorCallback(e) {
         // If we can't get the network info we should still tell Cordova
         // to fire the deviceready event.
         if (channel.onCordovaConnectionReady.state !== 2) {
             channel.onCordovaConnectionReady.fire();
         }
         console.log("Error initializing Network Connection: " + e);
-    });
+    }
+
+    exec(successCallback, errorCallback, "NetworkStatus", "getConnectionInfo", []);
+};
+
+var me = new NetworkConnection();
+var timerId = null;
+var timeout = 500;
```

Comment:
jest to kod z oryginalnej linii 53 (wyżej). opóźnia wysłanie informacji o statusie offline do apki.